cmake/onnxruntime_unittests.cmake

```diff
@@ -857,7 +857,7 @@ if (onnxruntime_BUILD_WEBASSEMBLY)
   endif()
 endif()
 
-target_link_libraries(onnx_test_runner PRIVATE onnx_test_runner_common ${GETOPT_LIB_WIDE} ${onnx_test_libs})
+target_link_libraries(onnx_test_runner PRIVATE onnx_test_runner_common ${GETOPT_LIB_WIDE} ${onnx_test_libs} nlohmann_json::nlohmann_json)
```

Comment:
Just out of curiosity: Was `nlohmann_json::nlohmann_json` added because updated json submodule?

Reply:
No, because I need to use it to parse `onnx_backend_test_series_overrides.jsonc` in `onnx_test_runner`. See `main.cc`.